CHANGELOG.md

```diff
@@ -42,6 +45,59 @@
     * `using ListInstancesRange = google::cloud::internal::PaginationRange`
     * `using ListInstanceConfigsRange = google::cloud::internal::PaginationRange`
 
+### Bigtable
+
+* feat(bigtable): create logging layer for Cloud Bigtable (#5515)
+* docs: document Bigtable thread safety (#5394)
+
+### Pub/Sub
+
+* feat(pubsub): by default Subscriber::Subscribe retries forever (#5552)
+* fix(pubsub): deadlock during streaming pull shutdown (#5547)
+* fix(pubsub): deadlock during fire & forget shutdown (#5541)
+* refactor!: PaginationRange<T> is now an alias to StreamRange<T> (#5538)
+* fix(pubsub): treat some kUnavailable as success (#5510)
```

Comment:
That was removed as part of #5552, I would remove it, but up to you.

Reply:
Done.